code/game/sound.dm

```diff
@@ -297,6 +297,8 @@
 			if("bone_break")
 				S = pick('sound/effects/bone_break1.ogg','sound/effects/bone_break2.ogg','sound/effects/bone_break3.ogg','sound/effects/bone_break4.ogg','sound/effects/bone_break5.ogg','sound/effects/bone_break6.ogg','sound/effects/bone_break7.ogg')
 			//misc mobs
+			if("pred_pain")
+				S = pick('sound/voice/pred_pain1.ogg','sound/voice/pred_pain2.ogg','sound/voice/pred_pain3.ogg','sound/voice/pred_pain4.ogg','sound/voice/pred_pain_rare1.ogg')
```

Comment:
you need to use weights in this pick() like the male_pain pick() to make the rare pain actually rare